### PR TITLE
Added docs to the NFT module

### PR DESCRIPTION
**Pull request by miladz68**
This change is [Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412)

### CODE REVIEW

**Review by wojtek-coreum — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412)* status: all files reviewed, 1 unresolved discussion (waiting on @dzmitryhil, @miladz68, @silverspase, and @ysv)

___
*[`x/asset/nft/spec/README.md` line 24 at r1](https://reviewable.io/reviews/CoreumFoundation/coreum/412#-NPHwGD39ckzhCN1TnHo:-NPHwGD39ckzhCN1TnHp:bigasfo) ([raw file](https://github.com/CoreumFoundation/coreum/blob/dcc5ea7e3bd2b609a2bd554ec0a9e07d51db983d/x/asset/nft/spec/README.md#L24)):*
> ```Markdown
> NFT tokens come with a set of features that the issuer can specify at the time of issuing a class, and then in some cases configured on each NFT level later.
> 
> here is the list of features:
> ```

here -> Here

**Review by miladz68 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @dzmitryhil, @silverspase, @wojtek-coreum, and @ysv)

___
*[`x/asset/nft/spec/README.md` line 24 at r1](https://reviewable.io/reviews/CoreumFoundation/coreum/412#-NPHwGD39ckzhCN1TnHo:-NPRGk6R1hV-tz-xvNDj:b-896fix) ([raw file](https://github.com/CoreumFoundation/coreum/blob/dcc5ea7e3bd2b609a2bd554ec0a9e07d51db983d/x/asset/nft/spec/README.md#L24)):*
<details><summary><i>Previously, wojtek-coreum (Wojtek) wrote…</i></summary><blockquote>

here -> Here
</blockquote></details>

Done.

**Review by wojtek-coreum — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @dzmitryhil, @silverspase, and @ysv)

**Review by miladz68 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412)* status: all files reviewed, 1 unresolved discussion (waiting on @dzmitryhil, @silverspase, and @ysv)

___
*[`x/asset/nft/spec/README.md` line 31 at r2](https://reviewable.io/reviews/CoreumFoundation/coreum/412#-NPX9uOg55Pml9KQkZlR:-NPXKtTlCfC4WO6mncc8:b-896fix) ([raw file](https://github.com/CoreumFoundation/coreum/blob/a0cd0e3a0e786ca7f3ff905a988cbefb86a30d12/x/asset/nft/spec/README.md#L31)):*
<details><summary><i>Previously, dzmitryhil (Dzmitry Hil) wrote…</i></summary><blockquote>

Let's also add the information about the queries, that for the queries it's expected to use the original NFT module.
</blockquote></details>

Done.

File: x/asset/nft/spec/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+If this feature is enabled, then for any user to receive any NFT of that class, they must be whitelisted to 
+receive that specific NFT. It follows that this feature allows the issuer of the class to whitelist an 
+account to hold an specific NFT of that class, or remove an account from whitelisted accounts for that NFT. 
+
+### Disable Sending
+If this feature is enabled, then the NFT cannot be directly transferred between users, meaning that user A cannot
+send the tokens they hold directly to user B. This feature opens up the door for different use cases, one of which is that it might be used to force transfer of ownership to go via DEX, so that the royalty fee is applied and the creator of the NFT always gets a royalty fee.
+
+### Royalty Rate
+This feature is related to the DEX, and if it is enable, every time that an NFT is traded on the DEX, a percentage of the the traded value is sent to the issuer as royalty fee.
```
Comment (pgoos) [inline x/asset/nft/spec/README.md:54]: typo: `if it is enable` -> `if it is enabled`

**Review by miladz68 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @silverspase and @ysv)

___
*[`x/asset/nft/spec/README.md` line 54 at r3](https://reviewable.io/reviews/CoreumFoundation/coreum/412#-NPHUHhlKzK_y2DaqnlI-r3-54:-NPaSX0t0esofFZWLkLJ:b-95txdl) ([raw file](https://github.com/CoreumFoundation/coreum/blob/798be397565df0ee2eacc27dfff307e080bb6bc7/x/asset/nft/spec/README.md#L54)):*
<details><summary><i>Previously, pgoos (Piotr Gesicki) wrote…</i></summary><blockquote>

typo: `if it is enable` -> `if it is enabled`
</blockquote></details>


Done

**Review by pgoos — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/coreum/412#-:-NQ6570i2wvXXpK66Ttw:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @silverspase and @ysv)

**Pull request merged**